### ISSUE TITLE
[Issue] Tons of errors config when loading hyprland

### ISSUE THREAD

**Issue by kaine0x00** [end-4/dots-hyprland#419]
- Linux distro: 
Arch Linux 
---
## The issue
After updating Hyprland, I am greeted with over a thousand lines of errors in my config files. Haven't really changed anything whatsoever lately, so what happened?
After trial and error, I figured out it had to do with commented lines. Apparently, any string of more than one # *##, ### or more* will be flagged as an error. Maybe the syntax for config files has been changed in Hyprland. 

Anyways, if anyone is having a similar problem, you may replace the multiple hashed comments in a file or files with the following sed command

``` bash
sed -i 's/##*/#/g' file1 file2 file3
```

**Comment by end-4** [comment]
but i thought this has been fixed already :(

**Comment by end-4** [comment]
can you show the error? which file is causing it?

**Comment by kaine0x00** [comment]
> but i thought this has been fixed already :(

Yeah, i basically re pulled the dotfiles from your repo and realised they didn't have multi-hashed comments. Only thing that troubled me were the custom configs which still had the problem and i couldn't just clone because they're my own. Solved that with the sed command earlier.

The error message i haven't got anymore, but I'm pretty sure it was something quite broad and generic like

```
error in line 1 /home/kaine/.config/hypr/hyprland/env.conf
error in line 2 /home/kaine/.config/hypr/hyprland/env.conf
. . .
```

TLDR: For the base end-4 configs just re pulling them from the repo and applying them should be fine. But for your custom configs you may need to use sed or any other text manipulation to remove multi-hash comment ocurrences.

keep up the good work! love the dots
